tap-snapshots/test/arborist/load-virtual.js.test.cjs

```diff
@@ -7298,9 +7304,106 @@ ArboristNode {
       "version": "5.5.0",
     },
     "tap" => ArboristNode {
+      "bundleDependencies": Array [
```

Comment:
are we meant to be listing meta deps as part of the bundleDependencies? i thought the intent was for this to only be the direct deps and everything beneath those is bundled implicitly and not listed here?

Reply:
Ah, this test is loading from a npm v6 lockfile.  In that case, we only know whether something is in the bundle or not, not who actually bundles it.  In order to avoid having to trace through the entire dep graph and find a minimal bundle set (which might be inaccurate anyway), we just build the node with an effective virtual `package.json` that bundles everything.

There was a bug where this extra bundle listing was done for _new_ lockfiles, resulting in unnecessary lockfile churn, but that's been fixed.

Reply:
ahhh ok :+1: